### ISSUE TITLE
Feature request: detect when a song cannot be played and auto-skip

### ISSUE THREAD

**Issue by itslenny** [pixelcrisis/turnstyles#43]
It's fairly common that a DJ is AFK and a song will come up that cannot be played and the room will just go silent foooorrreeevvveeerrrr. This should probably be something turntable (Deepcut) does and not part of this browser plugin, but I don't think they're adding any new features. So, it seems more likely to get it as a feature on Turnstyles which would still solve the problem as long as most AFK DJs are using this plugin.

<img width="1263" alt="Screenshot 2024-01-12 at 11 05 51 AM" src="https://github.com/pixelcrisis/turnstyles/assets/5210230/3d7227d4-82cb-4779-9e6a-b98c58f0ecff">

**Comment by NokaAngel** [comment]
While this is possible, I think a more likely issue depends on the Chrome extension's permissions, to allow this, since it would have to view the screen, and detect that this video is unavailable or blocked by XXX.
